### ISSUE TITLE
Crash IllegalAccessError 

### ISSUE THREAD

**Issue by ekager** [mozilla-mobile/focus-android#2417]
Don't have STR. It appears in GP and also have been able to trigger it with Monkey. 

```
2018-03-30 16:15:01.022 14940-14940/org.mozilla.focus.debug E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.mozilla.focus.debug, PID: 14940
    java.lang.IllegalAccessError: There's no active session with UUID f19478de-74f4-4a4a-a72f-543eaca1a021
        at org.mozilla.focus.session.SessionManager.getSessionByUUID(SessionManager.java:207)
        at org.mozilla.focus.fragment.UrlInputFragment.onCreate(UrlInputFragment.kt:130)
        at android.support.v4.app.Fragment.performCreate(Fragment.java:2246)
        at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1377)
        at android.support.v4.app.FragmentTransition.addToFirstInLastOut(FragmentTransition.java:1187)
        at android.support.v4.app.FragmentTransition.calculateFragments(FragmentTransition.java:1070)
        at android.support.v4.app.FragmentTransition.startTransitions(FragmentTransition.java:115)
        at android.support.v4.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2374)
        at android.support.v4.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2332)
        at android.support.v4.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2239)
        at android.support.v4.app.FragmentManagerImpl$1.run(FragmentManager.java:700)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6165)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:888)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:778)
```

**Comment by bbinto** [comment]
@ekager to confirm severity, STR etc for next triage.

**Comment by bbinto** [comment]
@ekager - do it now!

**Comment by sv-ohorvath** [comment]
I was able to reproduce the crash on v4.1. Verified as fixed on Beta v5.0. 
Device: Pixel XL (Android 8.1.0)
